### ISSUE TITLE
Expose property types for each property, e.g. `BackgroundColorProperty`

### ISSUE THREAD

**Issue by frenic** [frenic/csstype#68]
```ts
import { BackgroundColorProperty } from 'csstype';
const bgColor: BackgroundColorProperty = 'white';
```

Many users are using this instead of 

```ts
import { Properties } from 'csstype';
const bgColor: Properties['backgroundColor'] = 'white';
``` 

But only some property types that doesn't only include type `string` or `number` are exposed. Should all properties have an equivalent property type?

**Comment by frenic** [comment]
Default generic `string | 0` should also be applied to the `TLength` generic on each `type xProperty<TLength>`

**Comment by frenic** [comment]
Fixed in #78
